src/common/com/intellij/plugins/haxe/ide/generation/CreateGetterSetterFix.java

```diff
@@ -69,10 +65,14 @@ protected String buildFunctionsText(HaxeNamedComponent namedComponent) {
     final String typeText = HaxePresentableUtil.buildTypeText(namedComponent, ((HaxeVarDeclarationPart)namedComponent).getTypeTag());
     final StringBuilder result = new StringBuilder();
     if (myStratagy == Strategy.GETTER || myStratagy == Strategy.GETTERSETTER) {
-      buildGetter(result, namedComponent.getName(), typeText);
+      if (null == myHaxeClass.findHaxeMethodByName(HaxePresentableUtil.getterName(namedComponent.getName()))) {
```

Comment:
this eliminates duplicate getter / setter 
if one of them is already present and user regenerates (so other missing one could be generated) - the existing one should not be regenerated